lib/decorators.js

```diff
@@ -21,6 +21,13 @@ class Flags extends ValuesDecorator {
 const createFlags = (...values) => new Flags(values);
 createFlags.of = enumDomain => new Flags(null, enumDomain);
 
+class List extends ValuesDecorator {
```

Comment:
I'd prefer to make it a standalone class or create a new base class `CategoryDecorator` because it is a bit counterintuitive to store `category` in `this.values`.